tests/test_triangulation.py

```diff
@@ -44,18 +57,24 @@ def test_reproject(minimal_session):
     assert p2d.shape[-1] == 2
 
 
-def test_load_tracks(minimal_session):
-    cams = [x for x in Path(minimal_session).iterdir() if x.is_dir()]
-    p2d = load_tracks(minimal_session)
+def test_load_tracks(minimal_session, frames=(25, 75), excluded_views=("side")):
+    p2d = load_tracks(minimal_session, frames, excluded_views)
+    cams = [
+        x.name
+        for x in Path(minimal_session).iterdir()
+        if x.is_dir() and x.name not in excluded_views
+    ]
     assert p2d.shape[0] == len(cams)
+    assert p2d.shape[1] == frames[1] - frames[0]
     assert p2d.shape[-1] == 2
 
 
-def test_load_view(minimal_session):
+def test_load_view(minimal_session, frames=(25, 75)):
```

Comment:
Same as above -- arguments to testing functions should not be used to specify constants.

Reply:
resolved